contrib/fedora/cjdns.dyn.patch

```diff
@@ -0,0 +1,57 @@
+diff -up ./node_build/make.js.dyn ./node_build/make.js
+--- ./node_build/make.js.dyn	2015-11-02 17:59:41.000000000 -0500
++++ ./node_build/make.js	2015-11-04 19:57:49.961155943 -0500
+@@ -238,41 +238,9 @@ Builder.configure({
```

Comment:
This patch is for fedora *only* and only for tagged release v17.1.  It is not intended for general consumption.  It works for 17.1, and will be adjusted for subsequent tagged releases.  The cjdns.spec file that accompanies this patch compiles 17.1 - so the patch is guaranteed to apply to that version.  If you change the version, you might (or might not) need to adjust the patches.  The goal of this system is to keep the fedora specific changes (like changing the uid from nobody to cjdns) isolated in the contrib/fedora directory.